docs/cicd/others-script-ci-template.sh

```diff
@@ -0,0 +1,22 @@
+# This is just an example workflow for continous integration.
+# You should customize it to meet your own requirements.
+
+# Setup environment.
+# The `apt install` command is supposed to run inside the latest ubuntu system.
+# If you're using other platforms, please customize the command to set up your environment.
+sudo apt install -y nodejs npm
+
+# Checkout the code.
+# Adapt the placeholders to meet your needs.
+git clone {RepositoryEndpoint}
+cd {FolderName}
+
+# Build the project.
+# The way to build the current project depends on how you scaffold it.
+# Different folder structures require different commands set.
+cd tabs && npm install && npm run build
+
+# Run unit test.
+# Currently, no opinioned solution for unit test provided during scaffolding, so,
```

Comment:
we should just drop in a default test framework like jest

Reply:
or mocha

Reply:
I guess the expected change is only about the comments. Could you please help modify the comments to call out jest or mocha? thanks.